tests/integration/component.go

```diff
@@ -411,4 +411,41 @@ func componentTests(args ...string) {
 			helper.CmdShouldPass("odo", "delete", cmpName, "--app", appName, "--project", project, "-f")
 		})
 	})
+
+	Context("successive odo push to check for existence of environment variables", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			context = helper.CreateNewContext()
+			originalDir = helper.Getwd()
+		})
+
+		JustAfterEach(func() {
+			helper.DeleteProject(project)
+			helper.DeleteDir(context)
+			os.RemoveAll(context)
+		})
+
+		It("should create a component with environment variables and the env vars should persist multiple odo push", func() {
+			componentName := helper.RandString(6)
+			appName := helper.RandString(6)
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+			helper.CmdShouldPass("odo", "component", "create", "nodejs", componentName, "--app", appName, "--project", project, "--context", context)
```

Comment:
Replace with
```
helper.CmdShouldPass("odo", append(args, "create", "nodejs", componentName, "--app", appName, "--project", project, "--context", context)...)
```
so that it can be executed by both component and sub component test